gradio/components/chatbot.py

```diff
@@ -222,6 +224,7 @@ def __init__(
             resizeable: If True, the component will be resizeable by the user.
             max_height: The maximum height of the component, specified in pixels if a number is passed, or in CSS units if a string is passed. If messages exceed the height, the component will scroll. If messages are shorter than the height, the component will shrink to fit the content. Will not have any effect if `height` is set and is smaller than `max_height`.
             min_height: The minimum height of the component, specified in pixels if a number is passed, or in CSS units if a string is passed. If messages exceed the height, the component will expand to fit the content. Will not have any effect if `height` is set and is larger than `min_height`.
+            editable: If True, the user can edit the text in the chatbot. Can also be set to "all", which allows the user to edit assistant messages as well.
```

Comment:
Since "all" is allowed, how about the type is`editable: Literal["user", "all"] | None = None`

Reply:
I didn't go this route because a lot of gr.Blocks() demos use chatbot as input to store history, and therefore by default, interactive would be true, so these chatbots would become editable.

Reply:
I see, ok that makes sense, I agree with @freddyaboulton's suggestion above 👍